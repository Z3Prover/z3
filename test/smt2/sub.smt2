
; Copyright (c) 2015 Microsoft Corporation

(declare-const a Int)
(declare-const b Int)

(set-option :pp.flat-assoc false)

(simplify (- 8 4 1))
(display (- 8 4 1))
(display (- a b 10 a))
(display (- 8.0 4.0 1.0))

