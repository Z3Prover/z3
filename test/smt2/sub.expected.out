3
(- (- 8 4) 1)
(- (- (- a b) 10) a)
(- (- 8.0 4.0) 1.0)
